tests/integrate/Autotest.sh

```diff
@@ -100,6 +100,8 @@ check_out(){
     #------------------------------------------------------
     # check every 'key' word
     #------------------------------------------------------
+    ifail=0
+    ifatal=0
     for key in $properties; do
     
         if [ $key == "totaltimeref" ]; then
```

Comment:
Is it possible to add some "notes or explanations" in Autotest.sh? Otherwise only a small number of people can handle this script, others do not understand.